.github/workflows/master.yml

```diff
@@ -132,12 +132,18 @@ jobs:
         ./tests/db/compose.sh build --build-arg DEPENDENCIES="$(python setup.py -q dependencies)"
     - name: Run tests
       run: |
+        set +e
+        err=0
+        trap 'err=1' ERR
+
         for service in $(./tests/db/compose.sh config --services | grep '^mlflow-')
         do
           # Set `--no-TTY` to show container logs on GitHub Actions:
           # https://github.com/actions/virtual-environments/issues/5022
           ./tests/db/compose.sh run --rm --no-TTY $service pytest tests/store/tracking/test_sqlalchemy_store.py tests/db
         done
+
+        test $err = 0
```

Comment:
Does this work ?
My local test this command work `test $vv -eq 0 ` but `test $err = 0` does not work

Reply:
and before this line, we should remove the `trap` on ERR ?

Reply:
What do you mean by `test $err = 0` does not work?

Reply:
`set -e` means exit fast. What I want is not exit fast. Am I missing something?

Reply:
Quick test

```
% cat test.sh
#!/usr/bin/env bash

err=0
trap 'err=1' ERR

# fail
foo

# works
echo "bar"

test $err = 0

                                                                                
@ test-err -- INSERT --
% ./test.sh
./test.sh: line 7: foo: command not found
bar
                                                                                
@ test-err -- INSERT --
% echo $?
1
```

Reply:
right, `test $err = 0` works. I just test with `test $err=0` lack of whitespace.

Reply:
> set -e means exit fast,

I meant add `set -e` at L145 place. (without it also fine)

Reply:
> I meant add set -e at L145 place. (without it also fine)

I don't think it's needed because `test $err = 0` is the last command.

Reply:
And this step doesn't affect other steps in the job.